specification/network/resource-manager/Microsoft.Network/stable/2020-05-01/virtualWan.json

```diff
@@ -5219,14 +5342,6 @@
           "$ref": "./network.json#/definitions/SubResource",
           "description": "Reference to the remote virtual network."
         },
-        "allowHubToRemoteVnetTransit": {
```

Comment:
why are these two properties being removed?

Reply:
@ravbhatnagar , some responses-
1. The behavior for the old API version should remain unchanged. The new behavior is only from api version 2020-05-01
2. When a user creates a virtual hub created using the new version, they will not not be able to add hubvnet connections as a part of it. When they do a get using old version, it will not have any hubvnet connections. The other way- when we do a GET using new API version for virtualhub created with old API and hubvnet connections, we'd stop seeing the new hubvnet connections here. The user is expected to use a list on the hubvnet connections. (Ditto for PUTs)
3. The 2 properties are internal to the virtual hub. They are managed internally and the code currently ignores whatever the user sends. They shouldn't have been exposed to the user in the first place. I'll update the commit log with more details.